src/System.Net.HttpListener/tests/HttpListenerAuthenticationTests.cs

```diff
@@ -13,6 +13,7 @@
 
 namespace System.Net.Tests
 {
+    [ConditionalClass(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
```

Comment:
It may be nice to have a reason for future readers eg
`[ConditionalClass(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))] // httpsys does not exist`

Up to you.

Reply:
Will add comments in next commit.